File: app/src/main/java/com/codekk/net/NetException.java
package com.codekk.net;

import com.codekk.utils.UIUtils;

/**
 * by y on 2017/5/16
 */

class NetException extends RuntimeException {

    NetException(int code, String message) {
        UIUtils.toast(code + "---" + message);
    }

}

File: app/src/main/java/com/codekk/net/NetException.kt
package com.codekk.net

import com.codekk.utils.UIUtils

/**
 * by y on 2017/5/16
 */

class NetException(code: Int, message: String) : RuntimeException() {

    init {
        UIUtils.toast(code.toString() + "---" + message)
    }

}